### PR TITLE
added Press Section to about us page -issue #1972

### LINKED ISSUE #1972: Add Press section in About Us page
Follow this design for the addition of the Press section in the About Us page:
https://www.figma.com/file/EMKcg0eTnOlr7bRZGFdnZu/Web-app-product-spec?node-id=1%3A2
Both Desktop and Mobile designs are there.

Go to Press Kit button needs to go to:
https://www.notion.so/fightpandemics/Press-Kit-7146b85a49c848ec8395c0cadf3371b5

### PR DESCRIPTION
**Pull request by chrislemus**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s): resolve issue #1972 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/pages/AboutUs.js [hunk 1]
```diff
@@ -377,6 +379,25 @@ const AboutUs = () => {
         </SupportersLogosContainer>
       </SupporterContainer>
 
+      <PressContainer>
+        <FlexBox direction="column" align="center">
+          <h1>{t("Press")}</h1>
```
Comment (wadehrarshpreet) [inline client/src/pages/AboutUs.js]: @chrislemus Please update the translation file for the corresponding ids. Use camelCase for Ids in en_US.json you will find `aboutUs` property, you can create your own property inside that...

**Review by wadehrarshpreet — CHANGES_REQUESTED** [review]

File: client/src/pages/AboutUs.js [hunk 2]
```diff
@@ -377,6 +379,25 @@ const AboutUs = () => {
...
+          <p>{t("If you're a member of the press, contact us at ")}</p>
```
Comment (Zaydme) [inline client/src/pages/AboutUs.js]: For translation, you will need to add keys yo `client/src/locales/translations/en_US.json`
(For this particular case inside "aboutUs") like what @wadehrarshpreet said

**Review by rabbi08 — APPROVED** [review]
Verified the fix on laptop and mobile, it is working fine. 

Refer below screenshot.
Screenshot 2020-11-29 at 12 11 35 PM
Screen Shot 2020-11-29 at 12 13 22 PM

**Comment by CamilaArias** [comment]
Other than the keys that need to be added LGTM, good work :D

**Review by Zaydme — APPROVED** [review]
LGTM! Thanks @chrislemus! 🙌

**Pull request merged**